server/grpc_service.go

```diff
@@ -513,13 +506,92 @@ func (s *GrpcServer) forwardTSO(stream pdpb.PD_TsoServer) error {
 	}
 }
 
+func (s *GrpcServer) forwardTSORequestWithDeadLine(
```

Comment:
Please add `defer logutil.LogPanic()` for the goroutine.

Reply:
added

Reply:
added